modules/server/main.tf

```diff
@@ -92,8 +92,9 @@ locals {
     }],
 
   }
-  network_interface = local.network_interface_base[var.server_private ? "private" : "public"]
-  missing_emails    = ((var.sendgrid_api_key != "") && (var.forseti_email_sender == "" || var.forseti_email_recipient == "") ? 1 : 0)
+  missing_emails             = ((var.sendgrid_api_key != "") && (var.forseti_email_sender == "" || var.forseti_email_recipient == "") ? 1 : 0)
+  network_interface          = local.network_interface_base[var.server_private ? "private" : "public"]
+  policy_library_sync_folder = "policy_library_sync"
```

Comment:
It might be good to make this a variable instead of a local, since it seems like it is just a string (vs. something with a local hash, for example).

Reply:
Good idea, I have updated this to use a variable instead.